### PR TITLE
Stabilize tests for new function parseDateTimeInJodaSyntax

### PR DESCRIPTION
**Pull request by rschu1ze**
failed due to time zone randomization

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by tavplubix — APPROVED** [review]

**Comment by tavplubix** [comment]
Fast test has failed

**Comment by rschu1ze** [comment]
I know ... looking into it.

File: tests/queries/0_stateless/02668_parse_datetime_in_joda_syntax.sql [hunk 1]
```diff
@@ -3,13 +3,12 @@
...
-select parseDateTimeInJodaSyntax('ad 1999', 'G YYYY') = toDateTime('1999-01-01');
-select parseDateTimeInJodaSyntax('Ad 1999', 'G YYYY') = toDateTime('1999-01-01');
-select parseDateTimeInJodaSyntax('AD 1999', 'G YYYY') = toDateTime('1999-01-01');
-select parseDateTimeInJodaSyntax('AD 1999', 'G yyyy') = toDateTime('1999-01-01');
-select parseDateTimeInJodaSyntax('AD 1999 2000', 'G YYYY yyyy') = toDateTime('2000-01-01');
-select parseDateTimeInJodaSyntax('AD 1999 2000', 'G yyyy YYYY') = toDateTime('2000-01-01');
+select parseDateTimeInJodaSyntax('AD 1999', 'G YYYY', 'UTC') = toDateTime('1999-01-01', 'UTC');
+select parseDateTimeInJodaSyntax('ad 1999', 'G YYYY', 'UTC') = toDateTime('1999-01-01', 'UTC');
+select parseDateTimeInJodaSyntax('Ad 1999', 'G YYYY', 'UTC') = toDateTime('1999-01-01', 'UTC');
+select parseDateTimeInJodaSyntax('AD 1999', 'G yyyy') = toDateTime('1999-01-01', 'UTC');
```
Comment (tavplubix) [inline tests/queries/0_stateless/02668_parse_datetime_in_joda_syntax.sql:9]:
```suggestion
select parseDateTimeInJodaSyntax('AD 1999', 'G yyyy', 'UTC') = toDateTime('1999-01-01', 'UTC');
```

File: tests/queries/0_stateless/02668_parse_datetime_in_joda_syntax.reference [hunk 2]
```diff
@@ -3,19 +3,17 @@
...
+select parseDateTimeInJodaSyntax('AD 1999', 'G YYYY', 'UTC') = toDateTime('1999-01-01', 'UTC');
 1
-select parseDateTimeInJodaSyntax('ad 1999', 'G YYYY') = toDateTime('1999-01-01');
+select parseDateTimeInJodaSyntax('ad 1999', 'G YYYY', 'UTC') = toDateTime('1999-01-01', 'UTC');
 1
-select parseDateTimeInJodaSyntax('Ad 1999', 'G YYYY') = toDateTime('1999-01-01');
+select parseDateTimeInJodaSyntax('Ad 1999', 'G YYYY', 'UTC') = toDateTime('1999-01-01', 'UTC');
 1
-select parseDateTimeInJodaSyntax('AD 1999', 'G YYYY') = toDateTime('1999-01-01');
+select parseDateTimeInJodaSyntax('AD 1999', 'G yyyy') = toDateTime('1999-01-01', 'UTC');
```
Comment (tavplubix) [inline tests/queries/0_stateless/02668_parse_datetime_in_joda_syntax.reference:12]:
```suggestion
select parseDateTimeInJodaSyntax('AD 1999', 'G yyyy', 'UTC') = toDateTime('1999-01-01', 'UTC');
```

**Pull request merged**